vignettes/including-mae-data-in-teal.Rmd

```diff
@@ -16,9 +16,9 @@ You can read more about `MultiAssayExperiment` data [here](https://www.bioconduc
 
 
 ## Example application
-The below present an example application including `MultiAssayExperiment` data.
+The example below represents an application including `MultiAssayExperiment` data.
 
-```{r}
+```{r, eval = FALSE}
```

Comment:
line 41 there's a space before the comma. And I think in line 39 we should refer to tmh specifically as modules which do work with MAE data